build/podman/pom.xml

```diff
@@ -81,7 +82,7 @@
                         <artifactId>quarkus-maven-plugin</artifactId>
                         <version>${quarkus.platform.version}</version>
                         <configuration>
-                            <skip>true</skip>
+                            <build.skip>${podman-disabled}</build.skip>
```

Comment:
This disabled the build only on openshift. Baremetal jobs will still be affected